### PR TITLE
Optimized all python script files(Fixed - Issue #266)

### PR DESCRIPTION
**Pull request by ankitsaxena21**
I optimized the python files in the scripts folder. They were including many blank white spaces, too long comments & statements and improper indentations. I updated the codes as per [the official python style guide](https://www.python.org/dev/peps/pep-0008/).
All the python scripts now follows the standard style guide of python.

This PR covers issue #<266>.

### CODE REVIEW

**Comment by mackowski** [comment]
Thank you for this PR but we will review it after we agree how we want to solve this in discussion in #266 issue

**Comment by ankitsaxena21** [comment]
Thanks for the response!
I'll wait for the conclusion of the discussion in #266 issue.

**Comment by mackowski** [comment]
@ThunderSon can you review?

File: scripts/Generate_RSS_Feed.py [hunk 1]
```diff
@@ -16,37 +16,47 @@
...
 for pull_request in pull_requests:
     # Take only merged PR
     if pull_request["merged_at"] is None:
         continue
-    # Convert merge date from 2019-08-25T06:36:35Z To Sun, 19 May 2002 15:21:36 GMT
+    # Convert merge date from 2019-08-25T06:36:35Z To Sun,
+    # 19 May 2002 15:21:36 GMT
     merge_date_src = pull_request["merged_at"]
-    merge_date_dst = datetime.strptime(merge_date_src, "%Y-%m-%dT%H:%M:%SZ").strftime("%a, %d %B %Y %H:%M:%S GMT")
+    merge_date_dst = datetime.strptime(merge_date_src, "%Y-%m-%dT%H:%M:%SZ")
```
Comment (mackowski) [inline scripts/Generate_RSS_Feed.py]: this is not valid python syntax

File: scripts/Generate_CheatSheets_TOC.py [hunk 2]
```diff
@@ -24,11 +24,15 @@
...
     index_file.write("\n")
-    index_file.write(cs_md_link_template % ("Index Proactive Controls", "IndexProactiveControls.md"))
-    index_file.write("\n")        
+    index_file.write(cs_md_link_template % ("Index Proactive Controls",
+                                            "IndexProactiveControls.md"))
+    index_file.write("\n")
     for cheatsheet in cheatsheets:
-        if cheatsheet != "Index.md" and cheatsheet != "IndexASVS.md" and cheatsheet != "IndexProactiveControls.md" and cheatsheet != "TOC.md":
-            cs_name = cheatsheet.replace("_"," ").replace(".md", "").replace("Cheat Sheet", "")
+        if cheatsheet != "Index.md" and cheatsheet != "IndexASVS.md" and
```
Comment (mackowski) [inline scripts/Generate_CheatSheets_TOC.py]: This is not valid python syntax

**Comment by mackowski** [comment]
I will close this PR since it breaks the build and these are only internal scripts, not something related to the project. 

Please reopen this PR if you fix it.

**Pull request closed without merging**